clients/client-go/creds.go

```diff
@@ -65,6 +66,8 @@ type Client struct {
 	HTTPClient ReducedHTTPClient
 	// Context that aborts all requests with this client
 	Context context.Context
+	// HTTPBackoffClient holds the HTTP backoff settings for retries
+	HTTPBackoffClient *httpbackoff.Client
```

Comment:
This is new functionality for general clients, right?  Should it be highlighted in the README?